lib/resources/dstu2/patient.yaml

```diff
@@ -218,15 +253,15 @@ fields:
       "maritalStatus": {
         "coding": [
           {
-            "system": "http://hl7.org/fhir/v3/NullFlavor",
-            "code": "UNK",
-            "display": "Unknown"
+            "system": "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus",
+            "code": "S",
+            "display": "Never Married"
```

Comment:
Good change to have a working example here. :)

Reply:
I saw this a while back and like, whaaat?? 🙄